tensorflow_serving/model_servers/main.cc

```diff
@@ -203,26 +220,33 @@ int main(int argc, char** argv) {
   tensorflow::string model_base_path;
   bool use_saved_model = true;
   string platform_config_file = "";
+  string model_config_file;
   tensorflow::string model_version_policy =
       FileSystemStoragePathSourceConfig_VersionPolicy_Name(
           FileSystemStoragePathSourceConfig::LATEST_VERSION);
   std::vector<tensorflow::Flag> flag_list = {
       tensorflow::Flag("port", &port, "port to listen on"),
       tensorflow::Flag("enable_batching", &enable_batching, "enable batching"),
+      tensorflow::Flag("model_config_file", &model_config_file,
+                       "If non-empty, read an ascii ModelServerConfig "
+                       "protobuf from the supplied file name, and serve the "
+                       "models in that file. (If used, --model_name, "
+                       "--model_base_path and --model_version_policy "
+                       "are ignored.)"),
       tensorflow::Flag("model_name", &model_name, "name of model"),
+      tensorflow::Flag("model_base_path", &model_base_path,
+                       "path to export (required)"),
       tensorflow::Flag(
-          "model_version_policy", &model_version_policy,
```

Comment:
It would be great if you could eliminate this formatting change, because it makes it hard for us to merge on the google side. Thanks!